### PR TITLE
Fix compat typo

### PR DESCRIPTION
**Pull request by kescobo**
I accidentally set compat to

```
[compat]
DataFrames = "0.19, 0.20, 0.21, 22"
```

When I tagged the last version. Sorry for the complication :facepalm:

### CODE REVIEW

**Comment by kescobo** [comment]
Do I also need to do something on my repo end? I already updated the `Project.toml`, but do I need to change which commit that General points to?

File: M/Microbiome/Versions.toml [hunk 1]
```diff
@@ -29,4 +29,4 @@ git-tree-sha1 = "a89979a590578dd9dfb372757522dc881a84db76"
 git-tree-sha1 = "274452475b32b4dd54f9abb6e2297e88d5fe9004"
 
 ["0.6.5"]
-git-tree-sha1 = "4e901f0569d9684cc02130d489098cca084848d4"
+git-tree-sha1 = "8653f05a1a4ee373cf13792833801b6a47e98876"
```
Comment (fredrikekre) [inline M/Microbiome/Versions.toml:32]: You should not update this

Comment (kescobo) [reply M/Microbiome/Versions.toml:32]: Even though that sha points to something with a `Project.toml` saying it's compatible with DataFrames v22?

Comment (fredrikekre) [reply M/Microbiome/Versions.toml:32]: Yes, this is immutable. You can release 0.6.6 with the fix if you like. Can still merge this PR though, it shouldn't break anything.

Comment (kescobo) [reply M/Microbiome/Versions.toml]: OK, got it, thanks

**Pull request merged**